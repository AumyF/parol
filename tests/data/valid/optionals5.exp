%start Start

%%

/* 0 */ Start: "x" StartOpt /* Option */ "y";
/* 1 */ StartOpt: "a"; // Option<T>::Some
/* 2 */ StartOpt: ; // Option<T>::None
