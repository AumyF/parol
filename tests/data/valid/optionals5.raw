%start Start

%%

Start: "x" StartOpt /* Option */ "y";
StartOpt: "a"; // Option<T>::Some
StartOpt: ; // Option<T>::None
