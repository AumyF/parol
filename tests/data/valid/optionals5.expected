title: None
comment: None
start_symbol: Start
current_scanner: INITIAL
name: INITIAL;line_comments: [];block_comments: [];auto_newline_off: false;auto_ws_off: false;
Start: Alts(Alt(<0>T(x), O[Alts(Alt(<0>T(a)))], <0>T(y)));
